src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -1129,6 +1129,56 @@ TEST_F(HMICapabilitiesTest,
   EXPECT_FALSE(file_system::FileExists(file_cache_name_));
 }
 
+TEST_F(HMICapabilitiesTest,
+       OnCapabilityInitialized_RespondToAllPendingRAIRequestsIfTheyHold) {
+  MockApplicationManager mock_app_mngr;
+  NiceMock<event_engine_test::MockEventDispatcher> mock_dispatcher;
+  MockApplicationManagerSettings mock_application_manager_settings;
+
+  const std::string hmi_capabilities_file = "hmi_capabilities_sc2.json";
```

Comment:
@ydementieiev what does mean `sc2` ?

Reply:
@AByzhynar  **sc2 ->_second file with systemCapabilities_** but I not created it, only used after someone